### PR TITLE
Adnuntius Bid Adapter: Deal Id

### PR DESCRIPTION
**Pull request by mikael-lundin**
Added the option to pass on deals to prebid server. I had to comment out a line in the config file that caused an error with the validation. Please do let me know if I need to change anything in my PR. I tried force merging with prebid server master, but the validation error was still the same without me doing any changes.

### CODE REVIEW

File: adapters/adnuntius/adnuntiustest/supplemental/check-dealId.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+									"adUnits": [
+											{
+													"auId": "123",
+													"targetId": "123-test-imp-id",
+						"dimensions": [[300,250],[300,600]]
+											}
+									],
+									"context": "prebid.org",
+									"metaData": {}
+							}
```
Comment (gargcreation1992) [inline adapters/adnuntius/adnuntiustest/supplemental/check-dealId.json:45]: nitpick - formatting seems bit off for this json. See if you can correct it.

**Comment by gargcreation1992** [comment]
@mikael-lundin Friendly ping to please review the open comment.

**Comment by mikael-lundin** [comment]
Thanks, sorry was out traveling. Will address that right away!

**Comment by mikael-lundin** [comment]
I fucked up, will close this and open a new one.

**Pull request closed without merging**